README.md

```diff
@@ -282,6 +282,17 @@ Allow multiple statements in one query. While this allows batch queries, it also
 
 When `multiStatements` is used, `?` parameters must only be used in the first statement.
 
+##### `resultSetMetadata`
+
+```
+Type:           string
+Valid Values:   "full", "none"
+Default:        empty
+```
+
+Allow resultset metadata being optional. 
+By making resultset metadata transfer being optional, can potentially improve queries performance.
+
```

Comment:
Document that this requires MySQL 8.0+.

Add link to MySQL documentation: https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_resultset_metadata